providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdProvider.java

```diff
@@ -256,7 +256,7 @@ private void onReady() {
     }
 
     private void onError() {
-        log.info("Connection lost. Emit STALE event...");
+        log.debug("Stream error. Emitting STALE event and scheduling ERROR event...");
```

Comment:
Could we merge this with the line below?

Reply:
```java
        log.debug(
                "Stream error. Emitting STALE, scheduling ERROR, and waiting {}s for connection to become available.",
                gracePeriod);
```